src/screens/planActivity/PlanActivityScreen.tsx

```diff
@@ -98,14 +98,29 @@ export class PlanActivityScreen extends React.PureComponent<NavigationInjectedPr
     });
   };
 
+  shuffleDisabled(): boolean {
```

Comment:
There is a helper method `some` in a lodash library. You can use is to simplify the code and improve readibility.

```
shuffleDisabled(): boolean {
  const { planItemList } = this.state;
  if (!planItemList) {
    return true;
  }

  return some(planItemList, 'completed');
}
```

If I understand it correctly from current code `If some of the task are completed, the shuffle should be disabled`.